pages/funding.md

```diff
@@ -0,0 +1,129 @@
+    src="https://developers.google.com/open-source/gsoc/images/gsoc2016-sun-373x373.png"
+    title="Google Summer of Code" alt="Google Summer of Code"
+    style="display: inline; float: right; height: 4em; margin: 0
+    0.5em" /></a>
+
+The following contributors were sponsored to work on MDAnalysis through the [Google Summer of Code](https://summerofcode.withgoogle.com/) program.
+
+- **2024**: @ljwoods2, @lunamorrow, @talagayev
+- **2023**: @marinegor, @xhgchen 
+- **2022**: @aya9aladdin, @BFedder
+- **2021**: @ojeda-e, @orionarcher
+- **2020**: @cbouy, @hmacdope, @yuxuanzhuang
+- **2019**: @NinadBhat
+- **2018**: @ayushsuhane, @davidercruz
+- **2017**: @utkbansal
+- **2016**: @fiona-naughton, @jdetle
+
+<a href="https://developers.google.com/season-of-docs"> <img
+    src="https://developers.google.com/season-of-docs/images/SeasonofDocs_Icon_Grey_300ppi_trimmed.png"
+    title="Google Season of Docs" alt="Google Season of Docs"
+    style="display: inline; float: right; height: 4em; margin: 0 0.5em" /></a>
+
+The following technical writers were sponsored to work on MDAnalysis through the [Google Season of Docs](https://developers.google.com/season-of-docs) program.
+
+- **2019**: @lilyminium
+
+### [National Science Foundation](https://www.nsf.gov/)
+
+<a href="https://nsf.gov/">
+<img src="{{site.images}}/nsf.jpg" title="National Science
+    Foundation" alt="National Science Foundation" style="display:
+    inline; float: right; height: 4em; margin: 0 0.5em" /></a>
+ 
+Earlier work was partially supported by the NSF (as part of award ACI-1443054).
+
+- NSF DIBBS award, ACI-1443054 (**2014**): [CIF21 DIBBs: Middleware and High Performance Analytics Libraries for Scalable Data Science](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1443054) (**MDAnalysis Personnel**: @orbeckst)
+
+The following students were sponsored to work on MDAnalysis through the [NSF Research Experience for Undergraduates](https://www.nsf.gov/crssprgm/reu/) (REU) program.
+
+- **2021**: @ALescoulie, @edisj
+- **2020**: @edisj
+- **2019**: @nawtrey
+- **2018**: @hfmull
+- **2017**: @kaceyaurum
+- **2016**: @rbrtdlgd
+- **2015**: @ianmkenney
+
+### [NumFOCUS][]
+
+<a href="{{site.numfocus.sponsored_project}}"><img
+    src="{{site.images}}/numfocus-sponsored.png" title="NumFOCUS
+    sponsored project" alt="NumFOCUS Sponsored" style="display:
+    inline; float: right; height: 4em; margin: 0 0.5em" /></a>
+    
+MDAnalysis thanks NumFOCUS for its continued support as our fiscal sponsor, as well as through its [Small Development Grants](https://numfocus.org/programs/small-development-grants) (SDG) program.
+
+- SDG Round 3 (**2024**): Customizable and automated scientific molecular rendering with Molecular Nodes (**Personnel**: @yuxuanzhuang, @BradyAJohnston)
+- SDG Round 2 (**2023**): Unified and comprehensive documentation and learning resources for MDAnalysis (**Personnel**: @IAlibay, @jennaswa, @lilyminium, @micaela-matta, @orbeckst)
```

Comment:
Add @namiroues to Personnel.

I'd also reorder the list to better reflect that @jennaswa is PI, @namiroues is doing the work, and @micaela-matta is mentor:

```suggestion
SDG Round 2 (**2023**): Unified and comprehensive documentation and learning resources for MDAnalysis (**Personnel**: @jennaswa, @namiroues, @micaela-matta, @orbeckst, @IAlibay, @lilyminium)
```
```